shared/chat/conversation/container.js

```diff
@@ -57,9 +57,19 @@ class ConversationContainer extends Component<void, Props, State> {
   _onBack = () => {
     this.props.onBack()
   }
+  _onAttach = (inputs) => {
```

Comment:
Same as in the parent branch. Ultimately this entire wrapper should be able to go away or have greatly reduced props. Until that happens its props change a bunch all the time which trigger re-renders of the children as these dispatchMap props change. Instead of that, we make little local wrappers which won't change and pass those down instead. all short term